_includes/tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/harness-console-consumer-from-beginning.sh

```diff
@@ -0,0 +1 @@
+docker exec -it broker /usr/bin/kafka-console-consumer --topic example-topic --bootstrap-server broker:9092  --from-beginning --max-messages 8
```

Comment:
these `harness-XXX` console consumer files are needed so the console consumer will shut down during test harness run after it consumes the expected number of records.